crypto/BRCryptoWallet.h

```diff
@@ -203,6 +222,34 @@ extern "C" {
 
     DECLARE_CRYPTO_GIVE_TAKE (BRCryptoWallet, cryptoWallet);
 
+    /// MARK: Wallet Sweeper
+
+    typedef enum {
+        CRYPTO_WALLET_SWEEPER_SUCCESS,
+        CRYPTO_WALLET_SWEEPER_INVALID_OPERATION,
+        CRYPTO_WALLET_SWEEPER_INVALID_HASH
+    } BRCryptoWalletSweeperStatus;
+
+    extern BRCryptoWalletSweeper
+    cryptoWalletSweeperCreateAsBtc (BRCryptoKey key, BRCryptoAddressScheme scheme);
```

Comment:
Maybe given a `key` the sweeper does need more than one address?  That is, the UTXOs need to include LEGACY and SEGWIT public addresses?

Reply:
The address scheme is so that I know what our receive address will be.